src/swagger/index.js

```diff
@@ -23,6 +23,9 @@ const options = {
     servers: [
       {
         url: 'http://localhost:3000/api/v1',
+      },
+      {
+        url: 'https://blackninjas-backend-staging.herokuapp.com/api/v1',
```

Comment:
@shemaeric1234 May you differentiate these URLs i.e. property name? also instead of use env variables instead of hardcoding them.